docs/api.yml

```diff
@@ -118,11 +122,15 @@ paths:
           in: path
           required: true
           schema:
-            type: string
+            type: integer
 
       responses:
         201:
           description: The object has been created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OcflObjectVersion'
         400:
           description: The OcflObject was syntactically incorrect, for instance not all required fields were provided.
         401:
```

Comment:
the 409 conflict a few lines down is not needed if the service is not checking for conflicts